ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -1393,71 +1399,71 @@ func createPVMInstance(d *schema.ResourceData, client *st.IBMPIInstanceClient, i
 		ReplicantAffinityPolicy: flex.PtrToString(replicationpolicy),
 		Networks:                pvmNetworks,
 	}
-	if s, ok := d.GetOk(helpers.PIInstanceSSHKeyName); ok {
+	if s, ok := d.GetOk(Arg_KeyPairName); ok {
 		sshkey := s.(string)
 		body.KeyPairName = sshkey
 	}
 	if len(volids) > 0 {
 		body.VolumeIDs = volids
 	}
-	if d.Get(helpers.PIInstancePinPolicy) == "soft" || d.Get(helpers.PIInstancePinPolicy) == "hard" {
+	if d.Get(Arg_PinPolicy) == Soft || d.Get(Arg_PinPolicy) == Hard {
 		body.PinPolicy = models.PinPolicy(pinpolicy)
 	}
 
 	var assignedVirtualCores int64
-	if a, ok := d.GetOk(helpers.PIVirtualCoresAssigned); ok {
+	if a, ok := d.GetOk(Arg_VirtualCoresAssigned); ok {
 		assignedVirtualCores = int64(a.(int))
 		body.VirtualCores = &models.VirtualCores{Assigned: &assignedVirtualCores}
 	}
 
-	if st, ok := d.GetOk(helpers.PIInstanceStorageType); ok {
+	if st, ok := d.GetOk(Arg_StorageType); ok {
 		body.StorageType = st.(string)
 	}
-	if sp, ok := d.GetOk(PIInstanceStoragePool); ok {
+	if sp, ok := d.GetOk(Arg_StoragePool); ok {
 		body.StoragePool = sp.(string)
 	}
 
-	if dt, ok := d.GetOk(PIInstanceDeploymentType); ok {
+	if dt, ok := d.GetOk(Arg_DeploymentType); ok {
 		body.DeploymentType = dt.(string)
 	}
 
-	if ap, ok := d.GetOk(PIAffinityPolicy); ok {
+	if ap, ok := d.GetOk(Arg_AffinityPolicy); ok {
 		policy := ap.(string)
 		affinity := &models.StorageAffinity{
 			AffinityPolicy: &policy,
```

Comment:
Maybe rename this affinity variable to something like "affinityPolicy:. It's already competing names with the constant "Affinity."

Reply:
I don't think there is any, they are completely different from one another. See below.
`Arg_AffinityPolicy   = "pi_affinity_policy"` vs `Affinity  = "affinity"`